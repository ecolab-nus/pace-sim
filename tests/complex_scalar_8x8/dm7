0011000000110111010001010110001100110100011111001101111110101100
0011011110001001111001110111000010101010010100111010010011101000
0000010000110100110010110101001111100101010101110110100001101011
1111011111010110110101110100101010100000010101110000000111111011