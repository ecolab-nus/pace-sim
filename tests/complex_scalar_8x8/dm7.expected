1011011010101001100010101000001000111000100000011101010001111111
1100111101111111111111010111111111110001101010100111011110000001
1101101010000001010001101010101010110111100000101001100010101001
0011100101111111111010110111111110011001101010011000111010101010
